src/GraphQL.php

```diff
@@ -510,10 +510,41 @@ protected function getSchemaConfiguration($schema)
 
         $schema = is_array($schema) ? $schema : $this->schemas[$schemaName];
 
-        if (! is_string($schema)) {
+        return static::getNormalizedSchemaConfiguration($schema);
+    }
+
+    /**
+     * @return array<string, array|Schema>
+     */
+    public static function getNormalizedSchemasConfiguration(): array
+    {
+        return array_filter(array_map(function ($schema) {
+            try {
+                return static::getNormalizedSchemaConfiguration($schema);
+            } catch (SchemaNotFound $e) {
+                return null;
+            }
+        }, config('graphql.schemas')));
+    }
+
+    /**
+     * @param  Schema|array<array>|string|null  $schema
+     * @return Schema|array<array>
+     */
+    protected static function getNormalizedSchemaConfiguration($schema)
```

Comment:
I suggest to stay consistent with the visibility as this goes together with `getNormalizedSchemasConfiguration`:
```suggestion
    public static function getNormalizedSchemaConfiguration($schema)
```